### PR TITLE
Config admin tools

### PR DESCRIPTION
**Pull request by QuarpT**
## What does this change?

Admin tool for finding parameter store configuration by a key substring. 

The AWS console UI only enables search by prefix with no wildcard support. This would mean you'd have to know all the apps using a parameter to find it successfully. A parameter could potentially have 8 different prefixes if it were overridden by all the frontend apps. 

## What is the value of this and can you measure success?

Developers find it easier to search parameter store configuration

## Screenshots

picture 8

## Tested in CODE?

No

### CODE REVIEW

**Comment by philmcmahon** [comment]
This LGTM 👍 maybe Thomas or Nic should also have a look

File: admin/app/views/appConfig.scala.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            var $form = $(this),
+                term = $form.find('input[name="property"]').val(),
+                url = "/config/parameter/" + term;
+
+            var eventGetParams = $.get(url);
+
+            eventGetParams.done(function(data) {
+                $("#status").text("Ready");
+                Object.keys(data).forEach(function(key) {
+                    $("#key-values").append("<p>"+key+" <b>:</b> "+data[key]+"</p>");
```
Comment (TBonnin) [inline admin/app/views/appConfig.scala.html]: Do we want to show the `value`. There is some security implication. Personnally I won't show it, or maybe only the first 2 chars

File: admin/app/views/appConfig.scala.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            event.preventDefault();
+
+            var $form = $(this),
+                term = $form.find('input[name="property"]').val(),
+                url = "/config/parameter/" + term;
+
+            var eventGetParams = $.get(url);
+
+            eventGetParams.done(function(data) {
+                $("#status").text("Ready");
```
Comment (TBonnin) [inline admin/app/views/appConfig.scala.html]: what does `Ready` mean here?

Comment (QuarpT) [reply admin/app/views/appConfig.scala.html]: it means, 'ready to run a search'

File: admin/app/views/appConfig.scala.html (hunk 1 above)
Comment (QuarpT) [reply admin/app/views/appConfig.scala.html]: Okay, I'll make the change to the endpoint

**Pull request merged**